--- a/components/helium_services/helium_services_helpers.cc
+++ b/components/helium_services/helium_services_helpers.cc
@@ -17,7 +17,7 @@
 
 namespace helium {
 
-#if BUILDFLAG(IS_MAC)
+#if BUILDFLAG(IS_MAC) || BUILDFLAG(IS_WIN)
 #if defined(ARCH_CPU_ARM_FAMILY)
 constexpr std::string cpu_arch = "arm64";
 #else
@@ -129,8 +129,12 @@ GURL GetBrowserUpdateURL(const PrefServi
     }
 
 #if BUILDFLAG(IS_MAC)
-    std::string path = base::StringPrintf(
-        "/updates/mac/appcast-%s.xml", cpu_arch);
+    std::string path =
+        base::StringPrintf("/updates/mac/appcast-%s.xml", cpu_arch);
+    return GetServicesBaseURL(prefs).Resolve(path);
+#elif BUILDFLAG(IS_WIN)
+    std::string path =
+        base::StringPrintf("/updates/win/appcast-%s.xml", cpu_arch);
     return GetServicesBaseURL(prefs).Resolve(path);
 #else
     return GetDummyURL();
